### PR TITLE
Adds MVC and migrations for package test loads.

### LINKED ISSUE #574: Add ability to do loads only in test environment
9cb14dc0-767c-11e6-9193-db36dec35f6c
The buttons make calls to:
- FTP List = vnd_get_gen_files_list.do_ftp_list
- AFS List = vnd_get_gen_files_list.do_afs_list
- FTP Get = vnd_ftp_files.VND_DO_FTP_GEN
- AFS Get = vnd_afs_files.VND_DO_AFS_GEN
- Load all to Symphony = VND_LOAD_DATA_ELOAD.DO_LOAD
- Test load start ... = VND_LOAD_DATA_ELOAD.DO_LOAD(false)
- show files = reads entries from vnd_runlog

### LINKED ISSUE #583: Public view of the Eloader webforms
I would like to be able to give acquisitions (maybe others - it could be open to all) access to view eloader packages using the webform admin. But I do not want them to be able to do anything other than view the list of packages and the package details.

### LINKED ISSUE #588: Show package files loaded
From the list of packages page, there needs to be a link for each package that shows a list of files loaded.
screen shot 2018-11-13 at 3 47 40 pm

### PR DESCRIPTION
**Pull request by shelleydoljack**
Fixes #574 
Fixes #588 
Fixes #583

### CODE REVIEW

**Comment by shelleydoljack** [comment]
@jgreben ready for review and merge.

**Comment by shelleydoljack** [comment]
Related shared_configs PRs:
https://github.com/sul-dlss/shared_configs/pull/765
https://github.com/sul-dlss/shared_configs/pull/795

File: config/routes.rb [hunk 1]
```diff
@@ -42,6 +42,10 @@
   resources :packages do
     put :activate, on: :member
     put :deactivate, on: :member
+    if Rails.env.development? || Rails.env.test?
```
Comment (jgreben) [inline config/routes.rb]: See if you can remove these routing conditions and handle it all with can-can + the pl/sql settings user permissions?

File: config/routes.rb [hunk 2]
```diff
@@ -75,6 +79,15 @@
   get 'by_location' => 'accession_number_updates#by_location'
   get 'by_resource_type' => 'accession_number_updates#by_resource_type'
 
+  get 'package_files/completed' => 'package_files#completed'
+
+  # only run eloader test loads on dev and test
+  if Rails.env.development? || Rails.env.test?
```
Comment (jgreben) [inline config/routes.rb]: Same as above

**Review by jgreben — CHANGES_REQUESTED** [review]

**Review by jgreben — APPROVED** [review]

**Pull request merged**